common/test/acceptance/tests/helpers.py

```diff
@@ -353,17 +353,29 @@ def is_404_page(browser):
     return 'Page not found (404)' in browser.find_element_by_tag_name('h1').text
 
 
-def create_multiple_choice_problem(problem_name):
+def create_multiple_choice_xml(correct_choice=2, num_choices=4):
     """
     Return the Multiple Choice Problem Descriptor, given the name of the problem.
```

Comment:
I think this docstring should say Multiple Choice Problem XML instead of Descriptor.